src/js_user.hpp

```diff
@@ -30,6 +30,7 @@
 #include <realm/object-store/sync/sync_session.hpp>
 #include <realm/object-store/sync/sync_user.hpp>
 #include <realm/object-store/sync/app.hpp>
+#include "js_types.hpp"
```

Comment:
Is this still needed?

Reply:
It looks like I can build without it --> will remove the line.